velox/dwio/common/tests/utils/E2EFilterTestBase.h

```diff
@@ -85,6 +85,14 @@ class E2EFilterTestBase : public testing::Test {
   void SetUp() override {
     rootPool_ = memory::memoryManager()->addRootPool("E2EFilterTestBase");
     leafPool_ = rootPool_->addLeafChild("E2EFilterTestBase");
+    // Check environment variable because `buck test` does not allow pass in
```

Comment:
you can do:

```
buck test velox/... -- --my-gflag
```

Reply:
No it does not work, this only works for `buck run`, but randomized test really shines when used with `--stress-runs` which are only available in `buck test`